exercises/bank-account/example.py

```diff
@@ -0,0 +1,44 @@
+import threading
+
+
+class BankAccount(object):
+    def __init__(self):
+        self.lock = threading.Lock()
```

Comment:
I think it would be a good idea to initialize `open` flag in constructor as well:
```python
self.open = False
```